spec/statement.dd

```diff
@@ -260,21 +260,21 @@ $(GNAME ElseStatement):
         initialized to the end of the $(I ThenStatement).)
 
 ---
-import std.regexp;
+import std.regex;
 ...
-if (auto m = std.regexp.search("abcdef", "b(c)d"))
+if (auto m = std.regex.matchAll("abcdef", "b(c)d"))
 {
-    writefln("[%s]", m.pre);      // prints [a]
-    writefln("[%s]", m.post);     // prints [ef]
-    writefln("[%s]", m.match(0)); // prints [bcd]
-    writefln("[%s]", m.match(1)); // prints [c]
-    writefln("[%s]", m.match(2)); // prints []
+    writefln("[%s]", m.pre);    // prints [a]
+    writefln("[%s]", m.post);   // prints [ef]
+    writefln("[%s]", m.hit[0]); // prints [b]
+    writefln("[%s]", m.hit[1]); // prints [c]
+    writefln("[%s]", m.hit[2]); // prints [d]
```

Comment:
The `std.regex` equivalent of `std.regexp`'s `.match` is `opIndex`, i.e. `m[1]` will be c.